src/pcre.js

```diff
@@ -1,5 +1,15 @@
 // Declare global variable CodeMirror to avoid warnings in JSHint
-/* global CodeMirror */
+/* global CodeMirror, define */
+
+(function (mod) {
+  if (typeof exports === "object" && typeof module === "object") // CommonJS
+    mod(require("codemirror/lib/codemirror"));
```

Comment:
Should it not be `../../lib/codemirror` instead of `codemirror/lib/codemirror` ? The idea here is to remain consistent with upstream CodeMirror modes, e.g. https://github.com/codemirror/codemirror5/blob/fc0440bc7563aead662492b0f1b0512637545442/mode/asn.1/asn.1.js#L6

Reply:
This would be true if this mode script would be part of the package "codemirror", but it isn't. So it is not possible/useful to use relative paths to access the elements in codemirror package.

So it is accessed by package name and path. CommonJS handles the loading ...